### PR TITLE
fix(Dialogs): avoid double execution of native dialogs

### PR DESCRIPTION
**Pull request by pladaria**
[WEB-1962](https://jira.tid.es/browse/WEB-1962)

### CODE REVIEW

File: src/dialog.tsx [hunk 1]
```diff
@@ -213,9 +213,16 @@ const NativeModalDialog = ({
         message,
         title,
     });
+    const isOpenRef = React.useRef(false);
 
     React.useEffect(() => {
-        showNativeDialog(paramsRef.current);
+        if (isOpenRef.current) {
```
Comment (marcoskolodny) [inline src/dialog.tsx:219]: Maybe you can add a comment here explaining why is this needed

**Review by marcoskolodny — APPROVED** [review]

File: src/dialog.tsx (hunk 1 above)
Comment (pladaria) [reply src/dialog.tsx:220]: ok, added

**Review by brtbrt — APPROVED** [review]

**Review by atabel — APPROVED** [review]
